runtime/altair/src/lib.rs

```diff
@@ -2098,14 +2216,6 @@ impl fp_rpc::ConvertTransaction<sp_runtime::OpaqueExtrinsic> for TransactionConv
 	}
 }
 
-use cfg_types::{
-	locations::Location,
-	pools::PoolNav,
-	tokens::{FilterCurrency, LocalAssetId},
-};
-pub use pallet_loans::entities::{input::PriceCollectionInput, loans::ActiveLoanInfo};
-use runtime_common::transfer_filter::PreNativeTransfer;
```

Comment:
NOTE: Moved to top such that all dependencies are imported in the header. Having two separate places messed up IntelliJ's import helper because it seems to add imports to the location which is closer in terms of LoC.

Reply:
Yeah, it comes from removing the runtime-api feature. There were some `use` statements below I forgot to go up